plugins/bulk-import-common/package.json

```diff
@@ -17,7 +17,6 @@
     "supported-versions": "1.28.4",
     "pluginId": "bulk-import",
     "pluginPackages": [
-      "@janus-idp/backstage-plugin-bulk-import-backend",
```

Comment:
Why does it remove this? Comparing to another [plugin](https://github.com/backstage/community-plugins/blob/main/workspaces/explore/plugins/explore-common/package.json#L8:L14), I see all of these are included.

Reply:
Now that I am thinking about this, it may have to do with the fact that we are not releasing this plugin.

Reply:
NO IDEA. It's :magic_wand:

Reply:
Nevermind, because it adds the common plugin [here](https://github.com/janus-idp/backstage-plugins/pull/1933/files#diff-fad243a84db06c7d2362d205b080d4b293823fe9a2a7914e4f03d353b0e7b95dR19-R20) in the `bulk import` frontend plugin.